gradle/ge.gradle

```diff
@@ -27,6 +31,25 @@ gradleEnterprise {
         obfuscation {
             ipAddresses { addresses -> addresses.collect { address -> "0.0.0.0"} }
         }
+
+        tag("JDK" + System.getProperty("java.version").find(/\d+/)) // major version
```

Comment:
For what it's worth, these JDK details are already captured in the [build scan](https://ge.apache.org/s/iria4ppalnbdo#infrastructure). This information will show up in build scan comparisons as well. But capturing as tags and value is nice if you want the ability to filter on these values.

Reply:
Yes; this is exactly why I added it.  It's a major piece of useful information that differentiates builds.